apps/api/v1/lib/validations/selected-calendar.ts

```diff
@@ -10,14 +10,13 @@ export const schemaSelectedCalendarBaseBodyParams = SelectedCalendarSchema;
 export const schemaSelectedCalendarPublic = SelectedCalendarSchema.omit({});
 
 export const schemaSelectedCalendarBodyParams = schemaSelectedCalendarBaseBodyParams
-  .partial({
+  .pick({
```

Comment:
Instead of `omit` and constantly having to update this schema based on new fields, let's just pick the fields we need for the body based on the docs.